ui/packages/components/src/RunDetailsV2/RunInfo.tsx

```diff
@@ -49,7 +49,6 @@ export function RunInfo({ app, cancelRun, className, fn, rerun, run }: Props) {
 
           <CancelRunButton disabled={Boolean(endedAt)} onClick={cancelRun} />
           <RerunButton onClick={() => rerun({ fnID: fn.id })} />
-          <Button label="Rerun in Dev Server" size="small" />
```

Comment:
Why remove this?

Reply:
It's a new feature and V0 of the run details is to achieve feature parity